docs/getting-started/components.mdx

```diff
@@ -0,0 +1,150 @@
+# Add Component
+
+To understand better, let's review a few tips before you can add your first component.
+
+## Components
+
+This documentation refers to widgets as components since design tools like Figma use this terminology to describe their reusable UI elements.
+
+Components in Figma might be resembled by one or multiple variants which are variations of a component.
+
```

Comment:
`Variant` should have a section in the Terminology page

Reply:
Done